### ISSUE TITLE
v1.1.0-2 does not run on Manjaro

### ISSUE THREAD

**Issue by tnilzon** [oguzhaninan/Stacer#358]
v1.1.0-2 does not run on Manjaro

>/usr/share/stacer/stacer
Segmentation fault (core dumped)

>System:
  Kernel: 5.5.8-1-MANJARO x86_64 bits: 64 
  Desktop: KDE Plasma 5.18.3 Distro: Manjaro Linux

**Comment by vasiliu4556789** [comment]
/usr/share/stacer/stacer
Segmentation fault (core dumped)

systemd-coredump[1284959]: Process 1273604 (gjs) of user 1000 dumped core.
                                                     
                                                     Stack trace of thread 1273604:
                                                     #0  0x00007fd73dbdf687 n/a (libgtk-3.so.0 + 0x351687)
                                                     #1  0x00007fd73da6afb7 n/a (libgtk-3.so.0 + 0x1dcfb7)
                                                     #2  0x00007fd7427417fe g_cclosure_marshal_VOID__FLAGSv (libgobject-2.0.so.0 + 0x327fe)
                                                     #3  0x00007fd7427320a0 g_signal_emit_valist (libgobject-2.0.so.0 + 0x230a0)
                                                     #4  0x00007fd7427336b0 g_signal_emit (libgobject-2.0.so.0 + 0x246b0)
                                                     #5  0x00007fd73d9cb760 n/a (libgtk-3.so.0 + 0x13d760)
                                                     #6  0x00007fd73d9cbe18 n/a (libgtk-3.so.0 + 0x13de18)
                                                     #7  0x00007fd73da664ed gtk_switch_set_state (libgtk-3.so.0 + 0x1d84ed)
                                                     #8  0x00007fd73da6acce n/a (libgtk-3.so.0 + 0x1dccce)
                                                     #9  0x00007fd73dc6350e n/a (libgtk-3.so.0 + 0x3d550e)
                                                     #10 0x00007fd7427320a0 g_signal_emit_valist (libgobject-2.0.so.0 + 0x230a0)
                                                     #11 0x00007fd7427336b0 g_signal_emit (libgobject-2.0.so.0 + 0x246b0)
                                                     #12 0x00007fd73da66244 gtk_switch_set_active (libgtk-3.so.0 + 0x1d8244)
                                                     #13 0x00007fd742746c49 g_object_setv (libgobject-2.0.so.0 + 0x37c49)
                                                     #14 0x00007fd742746dec g_object_set_property (libgobject-2.0.so.0 + 0x37dec)
                                                     #15 0x00007fd7425dc072 n/a (libgio-2.0.so.0 + 0x80072)
                                                     #16 0x00007fd74274d61a g_closure_invoke (libgobject-2.0.so.0 + 0x3e61a)
                                                     #17 0x00007fd74272e97e n/a (libgobject-2.0.so.0 + 0x1f97e)
                                                     #18 0x00007fd742731ab9 g_signal_emit_valist (libgobject-2.0.so.0 + 0x22ab9)
                                                     #19 0x00007fd7427336b0 g_signal_emit (libgobject-2.0.so.0 + 0x246b0)
                                                     #20 0x00007fd7425e3e23 n/a (libgio-2.0.so.0 + 0x87e23)
                                                     #21 0x00007fd74264d845 n/a (libgio-2.0.so.0 + 0xf1845)
                                                     #22 0x00007fd7427320a0 g_signal_emit_valist (libgobject-2.0.so.0 + 0x230a0)
                                                     #23 0x00007fd7427336b0 g_signal_emit (libgobject-2.0.so.0 + 0x246b0)
                                                     #24 0x00007fd7425e141f n/a (libgio-2.0.so.0 + 0x8541f)
                                                     #25 0x00007fd7425debcb n/a (libgio-2.0.so.0 + 0x82bcb)
                                                     #26 0x00007fd7427dbc05 g_main_context_invoke_full (libglib-2.0.so.0 + 0x6dc05)
                                                     #27 0x00007fd7425df951 n/a (libgio-2.0.so.0 + 0x83951)
                                                     #28 0x00007fd7253862df n/a (libdconfsettings.so + 0x72df)
                                                     #29 0x00007fd725386cc9 n/a (libdconfsettings.so + 0x7cc9)
                                                     #30 0x00007fd7425d786b n/a (libgio-2.0.so.0 + 0x7b86b)
                                                     #31 0x00007fd7425dbe71 n/a (libgio-2.0.so.0 + 0x7fe71)
                                                     #32 0x00007fd74274d61a g_closure_invoke (libgobject-2.0.so.0 + 0x3e61a)
                                                     #33 0x00007fd74272e97e n/a (libgobject-2.0.so.0 + 0x1f97e)
                                                     #34 0x00007fd742731ab9 g_signal_emit_valist (libgobject-2.0.so.0 + 0x22ab9)
                                                     #35 0x00007fd7427336b0 g_signal_emit (libgobject-2.0.so.0 + 0x246b0)
                                                     #36 0x00007fd742749b46 n/a (libgobject-2.0.so.0 + 0x3ab46)
                                                     #37 0x00007fd7427451f2 g_object_notify_by_pspec (libgobject-2.0.so.0 + 0x361f2)
                                                     #38 0x00007fd73da66254 gtk_switch_set_active (libgtk-3.so.0 + 0x1d8254)
                                                     #39 0x00007fd73da6abb1 n/a (libgtk-3.so.0 + 0x1dcbb1)
                                                     #40 0x00007fd73d9cde4e n/a (libgtk-3.so.0 + 0x13fe4e)
                                                     #41 0x00007fd7427320a0 g_signal_emit_valist (libgobject-2.0.so.0 + 0x230a0)
                                                     #42 0x00007fd7427336b0 g_signal_emit (libgobject-2.0.so.0 + 0x246b0)
                                                     #43 0x00007fd73d825270 n/a (libgdk-3.so.0 + 0x91270)
                                                     #44 0x00007fd73d833cec n/a (libgdk-3.so.0 + 0x9fcec)
                                                     #45 0x00007fd7427d8c04 n/a (libglib-2.0.so.0 + 0x6ac04)
                                                     #46 0x00007fd7427d958f g_main_context_dispatch (libglib-2.0.so.0 + 0x6b58f)
                                                     #47 0x00007fd7427db531 n/a (libglib-2.0.so.0 + 0x6d531)
                                                     #48 0x00007fd7427dc543 g_main_loop_run (libglib-2.0.so.0 + 0x6e543)
                                                     #49 0x00007fd742532a8d n/a (libffi.so.7 + 0x6a8d)
                                                     #50 0x00007fd74253201b n/a (libffi.so.7 + 0x601b)
                                                     #51 0x00007fd74290d735 n/a (libgjs.so.0 + 0x77735)
                                                     #52 0x00007fd74290f628 n/a (libgjs.so.0 + 0x79628)
                                                     #53 0x00007fd740c9bc3d n/a (libmozjs-68.so + 0x112c3d)
                                                     #54 0x00007fd740c91325 n/a (libmozjs-68.so + 0x108325)
                                                     #55 0x00007fd740c8148f n/a (libmozjs-68.so + 0xf848f)
                                                     #56 0x00007fd740c9cb36 n/a (libmozjs-68.so + 0x113b36)
                                                     #57 0x00007fd740c9cd66 n/a (libmozjs-68.so + 0x113d66)
                                                     #58 0x00007fd740d61d38 n/a (libmozjs-68.so + 0x1d8d38)
                                                     #59 0x00007fd740d62512 _ZN2JS8EvaluateEP9JSContextNS_6HandleINS_13StackGCVectorIP8JSObjectN2js15TempAllocPolicyEEEEERKNS_22ReadOnlyCompileOptionsERNS_10SourceTextIDsEENS_13MutableHandle>
                                                     #60 0x00007fd7428e62e0 n/a (libgjs.so.0 + 0x502e0)
                                                     #61 0x00007fd7428e2214 gjs_context_eval (libgjs.so.0 + 0x4c214)
                                                     #62 0x000056443784b849 n/a (gjs-console + 0x2849)
                                                     #63 0x00007fd7429b3023 __libc_start_main (libc.so.6 + 0x27023)

**Comment by DiegoMC9** [comment]
same here:
```
Stack trace of thread 18917:
                #0  0x00007f84596f9078 _ZN7QString20fromLocal8Bit_helperEPKci (libQt5Core.so.5 + 0x12c078)
                #1  0x00007f84597acce9 _ZN14QStandardPaths14findExecutableERK7QStringRK11QStringList (libQt5Core.so.5 + 0x1dfce9)
                #2  0x00000000004c5ae4 _ZN11CommandUtil12isExecutableERK7QString (stacer + 0xc5ae4)
                #3  0x000000000043e596 n/a (stacer + 0x3e596)
                #4  0x00000000004c67ad __libc_csu_init (stacer + 0xc67ad)
                #5  0x00007f84590a1fb0 __libc_start_main (libc.so.6 + 0x26fb0)
                #6  0x000000000043e789 _start (stacer + 0x3e789)

```

**Comment by BCNelson** [comment]
This has been fixed in version 1.1.0-3 of the AUR package.
